src/index.js

```diff
@@ -349,8 +363,11 @@ function getNeighbors(p) {
     if (!(node in edgeLabels) || !(other in edgeLabels[node]) || edgeLabels[node][other] === dir) {
       const dir = edgeLabels[other] && edgeLabels[other][node] ? edgeLabels[other][node] : null
       const entryCost = points[other].type === 'burn' ? 1 : 0
-      const flybyBoost = points[other].type === 'flyby' ? points[other].flybyBoost : 0
+      const flybyBoost = points[other].type === 'flyby' || 'venus' ? points[other].flybyBoost : 0
```

Comment:
```suggestion
      const flybyBoost = points[other].type === 'flyby' || points[other].type === 'venus' ? points[other].flybyBoost : 0
```

Reply:
Yea -- thanks for noticing. I think this is what was causing misbehaviour in my pathing.